src/mdacli/libcli.py

```diff
@@ -27,22 +26,20 @@ class KwargsDict(argparse.Action):
 
     def __call__(self, parser, namespace, value, option_string=None):
         """Call me."""
-        try:
-            if Path(value).exists():
-                # read dict from JSON file
-                with open(value, 'r') as fin:
-                    jdict = json.load(fin)
-
-            else:
+        if value.startswith("{") and value.endswith("}"):
+            try:
                 jdict = json.loads(value)
 
-        except json.decoder.JSONDecodeError as err:
-            raise json.decoder.JSONDecodeError(
-                "An error ocurred when reading "
-                f"{self.dest!r} argument.",
-                err.doc,
-                err.pos,
-                ) from None
+            except json.decoder.JSONDecodeError as err:
+                raise json.decoder.JSONDecodeError(
+                    "An error ocurred when reading "
+                    f"{self.dest!r} argument.",
+                    err.doc,
+                    err.pos,
+                    ) from None
```

Comment:
why from `None` and not `from err`? Does this avoid a doubled error raise?

Reply:
`from` avoids the double error raise. `from None` because there is no need to show the user the exception stack. (I believe)